f2/src/forms/MoneyLostInfoForm.js

```diff
@@ -169,6 +171,11 @@ export const MoneyLostInfoForm = (props) => {
                 )}
                 <Field
                   name="transaction"
+                  label={
```

Comment:
This is unnecessary, this field is too complicated. We don't need all the rows above. They can be replaced with label and helpText props. See WhenDidItHappen.js for the correct way to use the DatePicker.

Reply:
This section of code in previous implementation before this PR had been refactored and in the latest commit